concepts/loops/about.md

```diff
@@ -0,0 +1,178 @@
+# About
+
+In Java there are four looping constructs, two that are condition centric: `while` and `do-while` and the other two are iteration centric: `for` and `for-each`.
+
+## For
+
+A for loop provides a mechanism to execute a group of statements repeatedly until some condition is met.
```

Comment:
Ironically this sentence talks about `for` using a condition, where the first paragraph lists `while` and `do-while` as being condition-centric and `for`/`for-each` being something else 😜

Reply:
Done!